mantidimaging/core/data/dataset.py

```diff
@@ -218,15 +220,14 @@ def _get_stack_data_type(stack_id: uuid.UUID, dataset: Dataset) -> str:
         return "Recon"
     if stack_id in [stack.id for stack in dataset._stacks]:
```

Comment:
NITPICK: While we're touching this area of code, we should probably refactor how we access the `_stacks` to go through `stacks` to maintain encapsulation, understanding that is is a subclass of `Dataset`.

Reply:
I might save that for a future unification of how different stack types are handled.